criu/sk-tcp.c

```diff
@@ -431,7 +435,11 @@ int restore_one_tcp(int fd, struct inet_sk_info *ii)
 	pr_info("Restoring TCP connection\n");
 
 	if (opts.tcp_close &&
-		ii->ie->state != TCP_LISTEN && ii->ie->state != TCP_CLOSE) {
+	    ii->ie->state != TCP_LISTEN && ii->ie->state != TCP_CLOSE) {
```

Comment:
I met a case that "state = TCP_CLOSE" for a socket, probably an unconnected socket dumped by function `gen_uncon_sk`

In this case, shall we just return 0 here when `state == TCP_CLOSE`, although we don't need to call `shutdown`.

Currently, it falls to `restore_tcp_conn_state` function below, which will look for the "tcp-stream.img", and failed because we don't have that image.